stdlib/runtime/string.gr

```diff
@@ -531,7 +531,8 @@ let rec heapValueToString = (ptr, extraIndents, toplevel) => {
         let rparen = ")"
         let mut strings = [rparen]
         if (tupleLength <= 1n) {
-          // Special case: unary tuple
+          // Special case: unary tuple, which is not valid Grain syntax; however, boxed values
+          // are stored as a unary tuple, so we keep this in case one gets printed somehow
```

Comment:
You can print a box and it prints as a unary tuple. We should make an issue to have real printing for boxes.

Reply:
Is this message fine or do you want me to remove "somehow"?

Reply:
Yeah, we can just drop the somehow.
```suggestion
          // Special case: unary tuple, which is not valid Grain syntax; however, boxed values
          // are stored as a unary tuple, so we keep this in case one gets printed
```